components/contract-components/contract-release-form/landing-fieldset.tsx

```diff
@@ -0,0 +1,303 @@
+import { SelectOption } from "core-ui/batch-upload/lazy-mint-form/select-option";
+import { useImageFileOrUrl } from "hooks/useImageFileOrUrl";
+import { replaceIpfsUrl } from "lib/sdk";
+import { Dispatch, SetStateAction } from "react";
+import { useFormContext } from "react-hook-form";
+import { BsCode, BsEye } from "react-icons/bs";
+import { FiTrash, FiUpload } from "react-icons/fi";
+import {
+  Card,
+  FormErrorMessage,
+  FormHelperText,
+  FormLabel,
+  Heading,
+  Link,
+  Text,
+} from "tw-components";
+
+interface LandingFieldsetProps {
+  contractSelection: "standard" | "proxy" | "factory";
+  setContractSelection: Dispatch<
+    SetStateAction<"standard" | "proxy" | "factory">
+  >;
+  latestVersion: string | undefined;
+  placeholderVersion: string;
+}
+
+export const LandingFieldset: React.FC<LandingFieldsetProps> = ({
+  contractSelection,
+  setContractSelection,
+  latestVersion,
+  placeholderVersion,
+}) => {
+  const form = useFormContext<ExtraPublishMetadata>();
+  const logoUrl = useImageFileOrUrl(form.watch("logo"));
+
+  return (
+    <Flex gap={16} direction="column" as="fieldset">
+      <Flex gap={2} direction="column">
+        <Heading size="title.lg">
+          {!latestVersion ? "Publish" : "Edit"} your contract
+        </Heading>
+        <Text fontStyle="normal">
+          Publishing your contract makes it shareable, discoverable, and
+          deployable in a single click.{" "}
+          <Link
+            color="blue.500"
+            isExternal
+            href="https://portal.thirdweb.com/release"
+          >
+            Learn more
+          </Link>
+        </Text>
+      </Flex>
+      <Flex gap={6} w="full">
+        <FormControl isInvalid={!!form.formState.errors.logo} w="auto">
+          <FormLabel>Image</FormLabel>
+          <Box width={{ base: "auto", md: "141px" }}>
+            <FileInput
+              accept={{ "image/*": [] }}
+              value={logoUrl}
+              setValue={(file) => form.setValue("logo", file)}
+              border="1px solid"
+              borderColor="gray.200"
+              borderRadius="md"
+              transition="all 200ms ease"
+              _hover={{ shadow: "sm" }}
+              renderPreview={(fileUrl) => (
+                <Image
+                  alt=""
+                  w="100%"
+                  h="100%"
+                  src={replaceIpfsUrl(fileUrl)}
+                  borderRadius="full"
+                />
+              )}
+              helperText="logo"
+            />
+          </Box>
+          <FormErrorMessage>
+            {form.formState.errors?.logo?.message as unknown as string}
+          </FormErrorMessage>
+        </FormControl>
+        <Flex flexDir="column" gap={4} w="full">
+          <FormControl
+            isInvalid={!!form.formState.errors.displayName}
+            isRequired
+          >
+            <FormLabel>Release Name</FormLabel>
+            <Input {...form.register("displayName")} required />
```

Comment:
we should be using `register('displayName', { required: true })`

Reply:
fixed in https://github.com/thirdweb-dev/dashboard/pull/782/commits/4085423a51450a30723a9dad73401a21d3fdd846